packages/react-icons/build-verify.test.js

```diff
@@ -1585,6 +1627,151 @@ describe('Build Verification', () => {
     });
   });
 
+  describe(`Atoms`, () => {
+    function getAssetPaths() {
+      const svgPathEsm = path.join(__dirname, 'lib', 'atoms/svg');
+      const svgPathCjs = path.join(__dirname, 'lib-cjs', 'atoms/svg');
+      const fontsPathEsm = path.join(__dirname, 'lib', 'atoms/fonts');
+      const fontsPathCjs = path.join(__dirname, 'lib-cjs', 'atoms/fonts');
+      return { svgPathEsm, svgPathCjs, fontsPathEsm, fontsPathCjs };
+    }
+
+    /**
+     *
+     * @param {string} assetPath
+     */
+    async function getStats(assetPath) {
+      const files = await readdir(assetPath);
+      const jsFiles = files.filter((f) => f.endsWith('.js'));
+      return { files, jsFiles };
+    }
+
+    it(`should have same number of atoms/svg icon files in lib and lib-cjs`, async () => {
+      const { svgPathCjs, svgPathEsm } = getAssetPaths();
+      const esmStats = await getStats(svgPathEsm);
+      const cjsStats = await getStats(svgPathCjs);
+      expect(esmStats.jsFiles.length).toMatchInlineSnapshot(`2764`);
```

Comment:
snapshot on purpose so it can be updated during release to not leave repo in invalid state